src/modules/users/dto/user.dto.ts

```diff
@@ -0,0 +1,11 @@
+export class CreateUserDto {
+  uid?: string;
```

Comment:
uid field is not needed on create form to create a User, Prisma auto-generates one. (Remove this optional field)